<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conectar</class>
 <widget class="QWidget" name="Conectar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio do jogo</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string>background-color:white;</string>
  </property>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancela">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>461</width>
     <height>230</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>118</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Seu nome:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_nome"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_servir">
      <property name="text">
       <string>Ser o servidor</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Quantos Clientes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_n_clientes">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMask">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_cliente">
      <property name="text">
       <string>Ser um cliente</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ip do Servidor:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_ip_servidor">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_single">
      <property name="text">
       <string>Jogar sozinho</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background: url(633889786990781250.png) no-repeat;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edit_nome</tabstop>
  <tabstop>radio_servir</tabstop>
  <tabstop>edit_n_clientes</tabstop>
  <tabstop>radio_cliente</tabstop>
  <tabstop>edit_ip_servidor</tabstop>
  <tabstop>btn_ok</tabstop>
  <tabstop>btn_cancela</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancela</sender>
   <signal>clicked()</signal>
   <receiver>Conectar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_servir</sender>
   <signal>toggled(bool)</signal>
   <receiver>edit_n_clientes</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_cliente</sender>
   <signal>toggled(bool)</signal>
   <receiver>edit_ip_servidor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
